backend/migrations/2023-03-09-194135_plant_relations/up.sql

```diff
@@ -0,0 +1,70 @@
+-- Your SQL goes here
+CREATE TABLE genus (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR NOT NULL,
+    created_at TIMESTAMP DEFAULT now() NOT NULL,
+    updated_at TIMESTAMP DEFAULT now() NOT NULL,
+    CONSTRAINT genus_name_key UNIQUE (name)
+);
+CREATE TABLE subfamily (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR NOT NULL,
+    created_at TIMESTAMP DEFAULT now() NOT NULL,
+    updated_at TIMESTAMP DEFAULT now() NOT NULL,
+    CONSTRAINT subfamily_name_key UNIQUE (name)
+);
+CREATE TABLE family (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR NOT NULL,
+    created_at TIMESTAMP DEFAULT now() NOT NULL,
+    updated_at TIMESTAMP DEFAULT now() NOT NULL,
+    CONSTRAINT family_name_key UNIQUE (name)
+);
+ALTER TABLE plant_detail
+ADD COLUMN genus_id INTEGER REFERENCES genus (id);
+ALTER TABLE plant_detail
+ADD COLUMN subfamily_id INTEGER REFERENCES subfamily (id);
+ALTER TABLE plant_detail
+ADD COLUMN family_id INTEGER REFERENCES family (id);
+CREATE TYPE RELATION_TYPE AS ENUM ('likes', 'dislikes');
+CREATE TYPE HIERARCHY_LEVEL_TYPE AS ENUM ('plant', 'genus', 'subfamily', 'family');
+CREATE TABLE relations (
+    id SERIAL PRIMARY KEY,
+    from_id INTEGER NOT NULL,
+    from_type HIERARCHY_LEVEL_TYPE NOT NULL,
+    to_id INTEGER NOT NULL,
+    to_type HIERARCHY_LEVEL_TYPE NOT NULL,
+    relation_type RELATION_TYPE NOT NULL,
+    created_at TIMESTAMP DEFAULT now() NOT NULL,
+    updated_at TIMESTAMP DEFAULT now() NOT NULL
+);
+/*
```

Comment:
What is the idea of the code below?

Reply:
the code was just a temporary suggestion/solution on how to populate and connect metatables to the `plant_detail` before the structure of the genus/subfamily/family files is fixed. Refer to my [answer](https://github.com/ElektraInitiative/PermaplanT/pull/89#issuecomment-1466148887)

Reply:
removed